golem.Rproj

```diff
@@ -1,8 +1,8 @@
 Version: 1.0
 
-RestoreWorkspace: Default
-SaveWorkspace: Default
-AlwaysSaveHistory: Default
+RestoreWorkspace: No
+SaveWorkspace: No
+AlwaysSaveHistory: No
```

Comment:
Ah, ok, no problem. Feel free to close the PR or I can do so. I did this one because the issue 329 was open and marked 'hacktoberfest', so I thought you wanted to move forward with it - would you consider closing that issue if you don't want it to be completed? I'd love to make a contribution to the repo, so please feel free to let me know if there are other good first contributions. Thanks!

Reply:
Ah sorry my comment wasn't clear ! 

I do want to merge the QuitChildProcessesOnExit change, but I think the others should stay default :)

Reply:
I've updated with that change, thanks again!